multiview/predict/ctclassifier.py

```diff
@@ -0,0 +1,362 @@
+# Copyright 2019 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .base import BaseCoTrainEstimator
+import numpy as np
+from sklearn.naive_bayes import GaussianNB
+from ..utils.utils import check_Xs
+from ..utils.utils import check_Xs_y_nan_allowed
+
+
+class CTClassifier(BaseCoTrainEstimator):
+    """
+    Co-Training Classifier
+
+    This class implements the co-training classifier similar to as described
+    in [1]. This should ideally be used on 2 views of the input data which
+    satisfy the 3 conditions for multi-view co-training (sufficiency,
+    compatibility, conditional independence) as detailed in [1].
+
+    Parameters
+    ----------
+    h1 : classifier object
```

Comment:
Is there a reason why these classifier objects are called h1 and h2? otherwise, cn u rename it so it's more clear

Reply:
changed, and included in base estimator for co-training